### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).

It also includes the following components:

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: _translator-files/po/es/copyright.po [hunk 1]
```diff
@@ -3,26 +3,31 @@
...
 "Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Weblate 4.15.1-dev\n"
 
 #. type: Hash Value: footer asio
 #: ../wiki/en/misc/copyright.yml:0
 #, no-wrap
 msgid "ASIO &reg; compatible - ASIO is a trademark and software of Steinberg Media Technologies GmbH"
 msgstr ""
+"ASIO &reg; compatible - ASIO es una marca comercial y software de Steinberg "
```
Comment (ann0see) [inline _translator-files/po/es/copyright.po:24]: We're not yet sure if we can translate this.

File: _translator-files/po/fr/Running-a-Server.po [hunk 2]
```diff
@@ -251,92 +247,87 @@ msgstr "_Pour mettre à niveau votre serveur, répétez simplement les étapes c
...
 msgid "Servers can be run in one of three modes (either at home or on a 3rd party host), depending on your needs."
-msgstr "Vous pouvez exécuter votre serveur de différentes manières (soit à la maison, soit sur un hébergeur tiers) :"
+msgstr "Les serveurs peuvent fonctionner selon l'un des trois modes suivants (à domicile ou sur un hôte tiers), en fonction de vos besoins."
 
 #. type: Title ###
 #: ../wiki/en/Running-a-Server.md:91
-#, fuzzy, no-wrap
+#, no-wrap
 msgid "Unregistered mode"
-msgstr "2. Non enregistré"
```
Comment (ann0see) [inline _translator-files/po/fr/Running-a-Server.po:274]: Probably this change also breaks some html anchors. 

This isn't specific for one language but all. 
Just saying to make translators aware.

**Review by ann0see — COMMENTED** [review]
Thanks

File: _translator-files/po/fr/Running-a-Server.po (hunk 2 above)
Comment (trebmuh) [reply _translator-files/po/fr/Running-a-Server.po:274]: I'm not sure what you're referring to here.
I am/was under the impression that I "fixed" a wrong translation here.
That said, is seem you're meaning some deeper things.
Could you please elaborate?

Comment (ignotus666) [reply _translator-files/po/fr/Running-a-Server.po:274]: @ann0see I had a quick look but couldn't find any anchors pointing to "Unregistered mode". In any case, if the anchor needs fixing in EN, that will make the string a fuzzy in all languages. Some anchors will get fixed in #899 though.

@trebmuh don't worry, it's not specific to FR and might not be an issue anyway.

**Pull request merged**